Resources/Prototypes/_ShitChap/Entities/Weapons/Melee/Nullrod.yml

```
@@ -0,0 +1,466 @@
+    - type: RequiresDualWield
+      whitelist:
+        tags:
+          - WeaponPistolJackal
+
+- type: entity
+  name: jackal
+  parent: WeaponPistolCasul
+  id: WeaponPistolJackal
+  description: The second of two huge pistols, capable of firing regenerating silver bullets. Incapable of firing without its sister gun.
+  components:
+    - type: Sprite
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)